server/templates/static/homepage.html

```diff
@@ -270,7 +270,7 @@ <h3>Water</h3>
           <a href="{{ chart_link }}">
             <img class="img-thumbnail" src="/images/home-water_withdrawal_temp.png" />
           </a>
-          <small class="caption"><a href="{{ chart_link }}">Water withdrawal for irrigation vs. projected temperature rise in California</a> (source: USGS, NASA)</small>
+          <small class="caption"><a href="{{ chart_link }}">Water withdrawal for irrigation vs. projected temperature rise across US counties</a> (source: USGS, NASA)</small>
```

Comment:
While making name tweaks, could we add "Water" to  "Withdrawal rates" in https://datacommons.org/tools/timeline?

Reply:
done